pokedex/data/csv/items.csv

```diff
@@ -227,14 +227,14 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
 226,charcoal,19,1000,30,
 227,dragon-fang,19,1000,70,
 228,silk-scarf,19,1000,10,
-229,up-grade,10,2000,30,
+229,upgrade,10,2000,30,
```

Comment:
Identifiers are supposed to be semi-stable. That's why we have identifiers at all instead of just using the names verbatim. Can we leave up-grade, stick, and the various -wings as they were? Unless there's a good reason to change them.

Reply:
Reverted this, stick, *-wings, and adventure-rules/ilimas-normalium-z as well